### ISSUE TITLE
used --dont-rename  and xmal

### ISSUE THREAD

**Issue by xpoy1** [0xd4d/de4dot#193]
use --dont-rename and  -strtok for dotNET_Reactor

**Comment by xpoy1** [comment]
Sorry I post it because wrong press "Enter".
The reason is didn't copy the resources to new executable.
Although, I got new problem, some class's valid member filed be renamed,  when the other be kept
Original,
// Token: 0x040002A8 RID: 680
		public TextBlock Award3Text;

		// Token: 0x040002A9 RID: 681
		public TextBlock Award3Low;

		// Token: 0x040002AA RID: 682
		public TextBlock Award2Text;

		// Token: 0x040002AB RID: 683
		public TextBlock Award2Low;

		// Token: 0x040002AC RID: 684
		public TextBlock Award1Text;

		// Token: 0x040002AD RID: 685
		public TextBlock Award1Low;

>>>>>
		// Token: 0x040002A8 RID: 680
		public TextBlock Award3Text;

		// Token: 0x040002A9 RID: 681
		public TextBlock textBlock_0;

		// Token: 0x040002AA RID: 682
		public TextBlock Award2Text;

		// Token: 0x040002AB RID: 683
		public TextBlock textBlock_1;

		// Token: 0x040002AC RID: 684
		public TextBlock Award1Text;

		// Token: 0x040002AD RID: 685
		public TextBlock textBlock_2;

This is the commandline, 
D:\de4dot_reactor\de4dot.exe D:\new\new\game86.exe --strtyp delegate --strtok 06000619 --dont-rename --preserve-tokens -v > D:\new\new\decodelog.txt

**Comment by 0xd4d** [comment]
You didn't use `--dont-rename` the 2nd time you used de4dot.

**Comment by xpoy1** [comment]
In honest I forgot how I unpacked the victim, when I meet the rename problem,  I had tried --dont-rename several time, didn't get the expect clean work well file. 

The point is some partial of the member properly kept, and some partial members be renamed to textbox_0, textBlock_0, like so on. The 2 types member both be used in same functions, so if some member be kept, then there must a way  to kept them all.
